src/org/intellij/erlang/compilation/ErlangPrepareDependenciesCompileTask.java

```diff
@@ -101,36 +102,24 @@ public ErlangModuleBuildOrders compute() {
   }
 
   public static void addPrepareDependenciesFailedMessage(CompileContext context) {
-    context.addMessage(CompilerMessageCategory.WARNING, "Failed to submit dependencies info to compiler. Parse transform failures may occur.", null, -1, -1);
+    context.addMessage(CompilerMessageCategory.WARNING, "Failed to submit dependencies info to compiler.", null, -1, -1);
   }
 
   @TestOnly
-  static ErlangModuleBuildOrderDescriptor getModuleBuildOrder(Module module) throws CyclicDependencyFoundException {
-    return getModuleBuildOrderInner(module);
+  static List<ErlangDependencyTreeNode> getModuleSortedDependenciesTree(Module module) throws CyclicDependencyFoundException {
```

Comment:
Please, work on methods naming. What is ModuleSortedDependenciesTree, considering it's a list of tree nodes?